packages/openapi-v3/test/unit/controller-spec/controller-decorators/request-body-decorators/primative-types.test.ts

```diff
@@ -0,0 +1,74 @@
+import {
+  ControllerSpec,
+  post,
+  requestBody,
+  getControllerSpec,
+} from '../../../../../';
+import {ContentObject, SchemaObject} from '@loopback/openapi-v3-types';
+import {Class} from '@loopback/repository';
+import {expect} from '@loopback/testlab';
+
+// I will add all shortcut decorators if we think they are valuable
+// considering we are changing to DI soon
+describe('requestBody decorator', () => {
+  context('array', () => {});
+  context('infers request body with primative type - ', () => {
+    let actualSpec: ControllerSpec;
+    let expectedContent: ContentObject;
+
+    it('number', () => {
+      class MyController {
+        @post('/greeting')
+        greet(@requestBody() name: number) {}
+      }
+      givenControllerMethod({type: 'number'}, MyController);
```

Comment:
the name here is wrong, renamed it to `assertRequestBodySpec` in the next commit